mm2src/coins/rpc_command/offline_keys.rs

```diff
@@ -0,0 +1,458 @@
+use crate::CoinProtocol;
+use crate::CoinsContext;
+use bitcoin_hashes::hex::ToHex;
+use bitcrypto::ChecksumType;
+use common::HttpStatusCode;
+use crypto::privkey::key_pair_from_seed;
+use derive_more::Display;
+use http::StatusCode;
+use keys::{AddressBuilder, AddressFormat, AddressPrefix, NetworkAddressPrefixes, Private};
+use mm2_core::mm_ctx::MmArc;
+use mm2_err_handle::prelude::*;
+use serde::{Deserialize, Serialize};
+use serde_json::Value as Json;
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub enum KeyExportMode {
+    #[serde(rename = "hd")]
+    Hd,
+    #[serde(rename = "iguana")]
+    Iguana,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct GetPrivateKeysRequest {
+    pub coins: Vec<String>,
+    pub mode: KeyExportMode,
```

Comment:
let's make mode optional, meaning it defaults to what is specified by `ctx.enable_hd()` ref.
https://github.com/KomodoPlatform/komodo-defi-framework/blob/7d51c5a6ab4390e4db8c4510db7012abe6cec8ad/mm2src/mm2_core/src/mm_ctx.rs#L228